src/test/resources/features/account.feature

```diff
@@ -24,13 +31,35 @@ Feature: Requests for Account endpoint
   @UpdateAccount
   Scenario Outline: Update a Account
     Given I build the pathParams and updated body to request
-      | name  | <nameAccount> |
+      | name          | <nameAccount>   |
+      | AccountNumber | <numberAccount> |
+      | Phone         | <phoneAccount>  |
     When I add this "/Account/{id}" endpoint and send the request with updated body
```

Comment:
Could be "I set the "/Account/{id}" endpoint and execute the request with the updated body"

Reply:
Good suggestion. I will change it.